server/src/visitors/YmlEnumVisitor.ts

```diff
@@ -31,6 +31,9 @@ export class YmlEnumVisitor extends YmlBaseVisitor {
         this.yenum.enrichWith(doc, this.enumName);
         this.completionProvider.addCompletionItem(this.yenum);
         this.yenum.setDefinitionLocation(node.start, node.stop, this.uri);
+        // Add enum to the definitions, because it looks a lot like a class.
+        this.definitions.addDefinition(this.yenum);
+        // Keep this for compatibility.
```

Comment:
Compatibility of what with what?

Reply:
_Go To Implementation_ was already working for `enum`. But when using _Go To Definition_ on enums as for classes since it wasn't working it felt weird. That's why I know have both.

Reply:
I don’t understand that part: “when using Go To Definition on enums as for classes since it wasn't working”.
I also don’t get if the initial “Go To Implementation” is a typo meant to be “… Declaration” or if it’s meant to be “Implementation”.
All in all, I still can’t answer the question of “compatibility of what with what”. Basically, what happens if you remove the line that follows that comment.

Reply:
If I remove the line that follows, users won't be able to use _Go To Implementation_ for enums − if they were using it.

Reply:
Then I’d simply say something like ‘For “Go to implementation”.’ xD dunno